src/overlays/actors/ovl_En_Slime/z_en_slime.c

```diff
@@ -45,146 +77,1170 @@ ActorInit En_Slime_InitVars = {
+    /* Goron punch    */ DMG_ENTRY(0, EN_SLIME_DMGEFF_BLUNT),
+    /* Sword          */ DMG_ENTRY(1, EN_SLIME_DMGEFF_NORMAL),
+    /* Goron pound    */ DMG_ENTRY(0, EN_SLIME_DMGEFF_BLUNT),
+    /* Fire arrow     */ DMG_ENTRY(2, EN_SLIME_DMGEFF_NORMAL),
+    /* Ice arrow      */ DMG_ENTRY(2, EN_SLIME_DMGEFF_FREEZE),
+    /* Light arrow    */ DMG_ENTRY(2, EN_SLIME_DMGEFF_LIGHT),
+    /* Goron spikes   */ DMG_ENTRY(1, EN_SLIME_DMGEFF_NORMAL),
+    /* Deku spin      */ DMG_ENTRY(1, EN_SLIME_DMGEFF_NORMAL),
+    /* Deku bubble    */ DMG_ENTRY(0, EN_SLIME_DMGEFF_BLUNT),
+    /* Deku launch    */ DMG_ENTRY(2, EN_SLIME_DMGEFF_NORMAL),
+    /* UNK_DMG_0x12   */ DMG_ENTRY(0, EN_SLIME_DMGEFF_STUN),
+    /* Zora barrier   */ DMG_ENTRY(1, EN_SLIME_DMGEFF_ELECTRIC),
+    /* Normal shield  */ DMG_ENTRY(0, EN_SLIME_DMGEFF_NORMAL),
+    /* Light ray      */ DMG_ENTRY(0, EN_SLIME_DMGEFF_NORMAL),
+    /* Thrown object  */ DMG_ENTRY(0, EN_SLIME_DMGEFF_BLUNT),
+    /* Zora punch     */ DMG_ENTRY(1, EN_SLIME_DMGEFF_NORMAL),
+    /* Spin attack    */ DMG_ENTRY(1, EN_SLIME_DMGEFF_NORMAL),
+    /* Sword beam     */ DMG_ENTRY(0, EN_SLIME_DMGEFF_NORMAL),
+    /* Normal Roll    */ DMG_ENTRY(0, EN_SLIME_DMGEFF_NORMAL),
+    /* UNK_DMG_0x1B   */ DMG_ENTRY(0, EN_SLIME_DMGEFF_NORMAL),
+    /* UNK_DMG_0x1C   */ DMG_ENTRY(0, EN_SLIME_DMGEFF_NORMAL),
+    /* Unblockable    */ DMG_ENTRY(0, EN_SLIME_DMGEFF_NORMAL),
+    /* UNK_DMG_0x1E   */ DMG_ENTRY(0, EN_SLIME_DMGEFF_NORMAL),
+    /* Powder Keg     */ DMG_ENTRY(1, EN_SLIME_DMGEFF_NORMAL),
 };
 
-// sColChkInfoInit
-static CollisionCheckInfoInit D_80A31B3C = { 1, 22, 35, 60 };
+static CollisionCheckInfoInit sColChkInfoInit = { 1, 22, 35, 60 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80A31B54[] = {
+static TexturePtr sEyeTextures[] = {
+    gChuchuEyeOpenTex,
+    gChuchuEyeHalfTex,
+    gChuchuEyeClosedTex,
+    gChuchuEyeHalfTex,
+};
+
+static InitChainEntry sInitChain[] = {
     ICHAIN_F32(gravity, -2, ICHAIN_CONTINUE),
     ICHAIN_F32(targetArrowOffset, 6000, ICHAIN_STOP),
 };
 
-#endif
+static s32 sTexturesDesegmented = false;
+
+static Color_RGBA8 sBubblePrimColor = { 255, 255, 255, 255 };
+static Color_RGBA8 sBubbleEnvColor = { 150, 150, 150, 0 };
+static Vec3f sBubbleAccel = { 0.0, -0.8, 0.0 };
+
+static Color_RGBA8 sPrimColors[] = {
+    { 255, 255, 255, 255 },
+    { 255, 255, 0, 255 },
+    { 255, 255, 200, 255 },
+    { 225, 200, 255, 255 },
+};
+
+static Color_RGBA8 sEnvColors[] = {
+    { 140, 255, 195, 255 },
+    { 50, 255, 0, 255 },
+    { 255, 180, 0, 255 },
+    { 255, 50, 155, 255 },
+};
+
+static Vec3f sLimbPosOffsets[EN_SLIME_LIMBPOS_COUNT] = {
+    { 2000.0, 2000.0, 0.0 }, { -1500.0, 2500.0, -500.0 }, { -500.0, 1000.0, 2500.0 },
+    { 0.0, 4000.0, 0.0 },    { 0.0, 2000.0, -2000.0 },
+};
+
+AnimatedMaterial* sSlimeTexAnim;
+
+void EnSlime_Init(Actor* thisx, PlayState* play) {
+    s32 reviveTimeSeconds;
+    s32 i;
+    EnSlime* this = THIS;
```

Comment:
```suggestion
void EnSlime_Init(Actor* thisx, PlayState* play) {
    EnSlime* this = THIS;
    s32 reviveTimeSeconds;
    s32 i;
```
Does this still match? It's better to have this at the top of the scope